README.md

```diff
@@ -7,6 +7,35 @@
 An Ansible role for installing the `cyhy_archive.sh` script from
 [cisagov/cyhy-core](https://github.com/cisagov/cyhy-core).
 
+## Pre-requisites ##
+
+In order to execute the Molecule tests for this Ansible role in GitHub
+Actions, a build user must exist in AWS. The accompanying Terraform
+code will create the user with the appropriate name and
+permissions. This only needs to be run once per project, per AWS
+account. This user can also be used to run the Molecule tests on your
+local machine.
+
+Before the build user can be created, the following profile must exist in
+your AWS credentials file:
+
+- `cool-terraform-backend`
```

Comment:
This role's Terraform is setup to run against the CyHy account and not the COOL multi-account configuration. I believe this profile should be changed to something more appropriate as a result.

Reply:
See cisagov/skeleton-ansible-role-with-test-user#22.